code/game/gamemodes/clock_cult/clock_scripture.dm

```diff
@@ -720,6 +720,20 @@ Judgement: 10 servants, 100 CV, and any existing AIs are converted or destroyed
 	one_per_tile = TRUE
 	sort_priority = 15
 
+/datum/clockwork_scripture/create_object/admire_avian
+	descname = "Summon a Clockwork Hawk"
+	name = "Avian Admiration"
+	desc = "The Justicar had an admiration for avians, encouraging servants to watch as they soared through the sky without any need of mechanical assistance. Some Sects dedicated their whole life to this teaching, and this is the only scripture remaining of this lost group."
+	invocations = list("Arise from the ashes, oh mighty phoenix!","To the heavens, from the sky, I call upon you faithful pet!")
+	channel_time = 80
+	required_components = list("belligerent_eye" = 2, "vanguard_cogwheel" = 1, "hierophant_ansible" = 1)
```

Comment:
In any case, the required components for this if it was actually a serious thing would be

    required_components = list("belligerent_eye" = 1, "vanguard_cogwheel" = 2)
    consumed_components = list("belligerent_eye" = 1, "vanguard_cogwheel" = 1)